src/OpenLoco/src/Vehicles/VehicleBogie.cpp

```diff
@@ -312,7 +312,7 @@ namespace OpenLoco::Vehicles
             reliabilityFactor /= 16;
 
             auto& prng = gPrng1();
-            int32_t randVal = (prng.randNext(65535) * reliabilityFactor / 2) / 65536;
+            int32_t randVal = (prng.randNext(65535) * (reliabilityFactor / 2)) / 65536;
```

Comment:
this fix causes replays to diverge but i checked and it is how it is meant to work so will record new masters.